srcup/api.py

```diff
@@ -103,4 +112,4 @@ async def get_project_id(watchdog_api: str,
         if req.status == 200:
             return await req.json()
         else:
-            raise Exception("No project with name {name} exists")
\ No newline at end of file
+            return -1  # project does not exist
```

Comment:
Returning `None` instead is a bit cleaner I think

Reply:
Done